tests/check/signed_video_helpers.c

```diff
@@ -134,6 +134,8 @@ create_signed_nalus_with_sv(signed_video_t *sv, const char *str, bool split_nalu
     } else {
       rc = signed_video_add_nalu_part_for_signing_with_timestamp(
           sv, item->data, item->data_size, &g_testTimestamp, true);
+      // The check is only performed for the complete NALU.
+      ck_assert(!signed_video_is_golden_sei(sv, item->data, item->data_size));
```

Comment:
I just realized that the nalu split is done explicitly, so this call should be done before the if-statement.